src/Features/CSharp/Portable/QuickInfo/CSharpSemanticQuickInfoProvider.cs

```diff
@@ -181,6 +183,30 @@ protected override NullableFlowState GetNullabilityAnalysis(SemanticModel semant
             return sourceText.GetSubText(new Text.TextSpan(span.Start, Math.Min(maxLength, span.Length))).ToString();
         }).ToImmutableArray();
 
-        return new OnTheFlyDocsElement(symbol.ToDisplayString(), symbolStrings, symbol.Language);
+        return new OnTheFlyDocsElement(symbol.ToDisplayString(), symbolStrings, symbol.Language, hasDocumentationComment);
+
+        bool HasDocumentationComment(ImmutableArray<SyntaxReference> references)
```

Comment:
when creating a tooltip in QuickInfoUtilities.CreateQuickInfoItemAsync, it looks like we know whether if contains  doc-comment, is it possible log there instead so we don't have to check the syntax?